code/modules/antagonists/gang/outfits.dm

```diff
@@ -0,0 +1,56 @@
+/datum/outfit/families_police/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
```

Comment:
I think this return is indented a bit too far/much?

Reply:
according to github thats the correct tab amount

Reply:
yep, 2 tab indentations